src/audio/rtnr/rtnr.c

```diff
@@ -843,7 +843,6 @@ static int rtnr_prepare(struct comp_dev *dev)
 {
 	struct comp_data *cd = comp_get_drvdata(dev);
 	struct comp_buffer *sinkb;
-	struct comp_buffer *sink_c;
```

Comment:
This code _is_ compiled in the IPC3 stub but I just tested that again and clang version 16 (and others) does not complain about unused variables in that function. Mysteries of compiler warnings...

- https://github.com/thesofproject/sof/pull/8571#discussion_r1426976620

RTNR is not compiled in app/stub_build_all_ipc4.conf, feel free to add there.